src/FriendLibraryPage.tsx

```diff
@@ -122,40 +122,51 @@ const FriendLibraryPage: React.FC = () => {
     });
   }, []);
 
-  return (
-    <PageLayout
-      header={
-        !user ? <h1>User Not Found</h1> : <h1>{user && user.name}'s Library</h1>
-      }
-      error={error}
-      loaded={isLoaded}
-    >
-      {user && <FriendshipStatusButton friend={user} onClick={setUser} />}
-      <LibrarySearchBar
-        onSearchChange={filterResults}
-        header="Search this Library"
-      />
-      <AutoTable
-        data={filteredBooks}
-        title={<h3>Books</h3>}
-        placeholder={
-          books ? (
-            <span>No search results found</span>
+  if (auth.credential.userId?.toString() === id) {
```

Comment:
@Hashiphoto It looks like the userId is coming in as a number and not a string despite being typed as a string.

Reply:
Oh. I can fix that in https://github.com/ParaLibrary/paralibrary-backend/pull/33 if you want?

Reply:
Sure!

Reply:
It has been fixed. I confirmed it sends the userId as a string on login. I've gone ahead and merged https://github.com/ParaLibrary/paralibrary-backend/pull/33